src/assets/css/fonts.scss

```diff
@@ -0,0 +1,7 @@
+@font-face {
+  font-family: "fontHelvetica";
+  src: url("https://pushdev-code.github.io/shoping-cart/public/fonts/helveticaneue_medium-webfont.woff2");
+  font-size: 14px;
```

Comment:
keep this font size as a property per element. Font size won't always be 14px.

Reply:
Done